data/json/items/containers/military.json

```diff
@@ -227,5 +227,45 @@
     "weight": "22 g",
     "volume": "480 ml",
     "pocket_data": [ { "pocket_type": "CONTAINER", "rigid": true, "max_contains_volume": "470 ml", "max_contains_weight": "2000 g" } ]
+  },
+  {
+    "id": "55gal_drum_rubber",
+    "type": "GENERIC",
+    "category": "container",
+    "name": { "str": "collapsible drum" },
+    "description": "Beige collapsible drum with water valve and D-Rings.  The text on the top clearly says \"POTABLE WATER ONLY\".",
+    "weight": "20 kg",
+    "volume": "10 L",
```

Comment:
10 litres in volume but can contain 200 litres of stuff? If this is to represent its collapsible nature, perhaps consider making a collapsed version, and an unfolded version, like with the cardboard boxes.

Reply:
for what? making an additional version sound like adding more micromanagement for nothing. It has a `rigid:false` anyway, which means full barrel would have 210L of volume as expected